src/devices/honeywell_cm921.c

```diff
@@ -279,8 +383,9 @@ static int parse_msg(bitbuffer_t *bmsg, int row, message_t *msg)
 
     // Checksum: All bytes add up to 0.
     int bsum = add_bytes(bb, num_bytes) & 0xff;
-    int checksum_ok = bsum == 0;
+    int checksum_ok = (bsum == 0 || bsum == 1) ? 1 : 0 ;   // bizarely, some packets have a csum of 1 rather than 0
```

Comment:
conditional over a bool. I.e. this is identical: `int checksum_ok = (bsum == 0 || bsum == 1);`